src/sparse_ir/kernel.py

```diff
@@ -246,7 +262,7 @@ def _compute(self, u_plus, u_minus, v):
         not_tiny = abs_v >= 1e-200
         denom = np.ones_like(abs_v)
         np.divide(abs_v, np.expm1(-abs_v, where=not_tiny),
-                  out=denom, where=not_tiny)
+                out=denom, where=not_tiny)
```

Comment:
Please revert this ... PEP 8 specifies decendents should be aligned with the opening bracket.
```suggestion
                  out=denom, where=not_tiny)
```